frontends/election-manager/src/app.test.tsx

```diff
@@ -63,9 +63,19 @@ const EITHER_NEITHER_CVR_DATA =
   electionWithMsEitherNeitherWithDataFiles.cvrData;
 const EITHER_NEITHER_CVR_FILE = new File([EITHER_NEITHER_CVR_DATA], 'cvrs.txt');
 
+const EITHER_NEITHER_CVR_TEST_DATA =
+  electionWithMsEitherNeitherWithDataFiles.cvrTestData;
+const EITHER_NEITHER_CVR_TEST_FILE = new File(
+  [EITHER_NEITHER_CVR_TEST_DATA],
+  'cvrs.txt'
+);
+
 const EITHER_NEITHER_SEMS_DATA =
   electionWithMsEitherNeitherWithDataFiles.semsData;
 
+const zeroReportTitle =
+  'Print the pre-election Unofficial Full Election Tally Report';
```

Comment:
`zeroReportButtonTitle` might be a more accurate name (or given my comment below, `ZERO_REPORT_BUTTON_TITLE`)